test/hotspot/jtreg/runtime/cds/appcds/aotClassLinking/WeakReferenceTest.java

```diff
@@ -89,7 +89,7 @@ public String[] vmArgs(RunMode runMode) {
                 return new String[] {
                     "-Xlog:gc,cds+class=debug",
                     "-XX:AOTInitTestClass=" + mainClass,
-                    "-Xlog:cds+map,cds+map+oops=trace:file=cds.oops.txt:none:filesize=0",
+                  //"-Xlog:aot+map,aot+map+oops=trace:file=cds.oops.txt:none:filesize=0",
```

Comment:
Why is this commented out?

Reply:
This produces a lot of output and slows down the test execution. I added this log when I was debugging the test. Since it's no longer needed (but it's impacted by this PR), I think we should delete it.

I deleted this line instead of commenting it out.